config/config.js

```diff
@@ -1,5 +1,6 @@
 var fs = require('fs');
-
+dotenv = require('dotenv');
```

Comment:
this should only happen once in the project, if you don't want a specific file you can change 

`npm start` to use `-r` (https://nodejs.org/api/cli.html#cli_r_require_module)

likeso:
`node -r dotenv/config ./bin/www`

if you do that, don't forget to change `npm dev` as well

Reply:
fixed